packages/flutter_test/lib/src/binding.dart

```diff
@@ -189,6 +189,32 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     debugDisableShadows = disableShadows;
   }
 
+  /// Deprecated. Will be removed in a future version of flutter.
+  ///
+  /// This property has been deprecated to prepare for Flutter's upcoming
+  /// support for multiple views and multiple windows.
+  ///
+  /// This represents a combination of a [TestPlatformDispatcher] and a singular
+  /// [TestFlutterView]. Platform specific test values can be set through
+  /// [WidgetTester.platformDispatcher] instead. When testing individual widgets
+  /// or single view applications, view specific test values can be set through
```

Comment:
per our discussion: These docs need to be updated to reflect that [WidgetTester.view] can also be used in multi-view situations if pumpWidget is used.

Reply:
Updated. Give it a look and let me know if it meets your expectations.